### ISSUE TITLE
The .octree file format seems to be different than the format in the repository ooc_svo_builder

### ISSUE THREAD

**Issue by Philipp-M** [Forceflow/liboctree#1]
Hey,
there seem to be differences in the format described in OctreeFile.h and the repo 'ooc_svo_builder'
This e.g. is the result of the header file with the current ooc_svo_builder:
```
#octreeheader 1
gridlength 1024
n_nodes 1746140
n_data 1325468
END
```
but reading the 'readHeader' method kind of expects:
```
#octree
grid 1024 1024 1024
node_count 1746140
data_count 1325468
data_size ?
END
```
I have rewritten the readHeader method so that it works with the 'old' format as well, though after i have rewritten it, I recognized 'data_size', where there is no equivalent in the 'old' format...
I can create a pull request if you want to, since I've done a few other code fixes so that it is compatible with other compilers than MVC as well (like gcc or clang)

**Comment by Forceflow** [comment]
Correct, I'm rewriting the liboctree format - I'm afraid the ooc_svo_builder repo and this repo are a bit out of sync on the matter.

**Comment by Philipp-M** [comment]
Hey, thanks for the answer,
Is it possible to get the projects somehow in sync (maybe as extra dev branch)?
Or is the new format not implemented yet in the ooc_svo_builder repo?
I would like to contribute (since I've currently working at my Bachelor thesis which is about out-of-core rendering of SVOs on the GPU), I have already written a simple header-based library for the old format.
I'm currently writing a converter to a compact format for streaming on the GPU.
